storybook/stories/API/chart/LineChart.stories.tsx

```diff
@@ -20,3 +23,28 @@ export const Simple = {
     data: pageData,
   },
 };
+
+export const PlayMoveBrush = {
```

Comment:
Love the story and the interaction. Yet I think this story is better fitting in the Brush API. Instead of adding a brush example to every chart type, please move this over to: 

`storybook/stories/API/cartesian/Brush.stories.tsx`